packages/taquito-ledger-signer/src/taquito-ledger-signer.ts

```diff
@@ -43,19 +44,6 @@ export class InvalidDerivationTypeError extends Error {
   }
 }
```

Comment:
shouldn't `InvalidDerivationTypeError` be moved to error.ts as well?

Reply:
was already moved an imported just missed clearing the declaration

Reply:
nvm see what i missed